libsolidity/ast/AST.h

```diff
@@ -624,6 +625,7 @@ class FunctionDefinition: public CallableDeclaration, public Documented, public
 		ImplementationOptional(_body != nullptr),
 		m_stateMutability(_stateMutability),
 		m_isConstructor(_isConstructor),
+		m_isOverriding(_isOverriding),
```

Comment:
maybe just `overriding`?

Reply:
wouldn't that be inconsistent with the existing naming `isContructor` and `isIndexed` etc?

Reply:
actually the coding style says that `is` prefixes should be avoided, but yeah, we have a lot of them here.